silkrpc/core/storage_walker_test.cpp

```diff
@@ -63,7 +63,7 @@ class DummyCursor : public silkrpc::ethdb::CursorDupSort {
         co_return;
     }
 
-    asio::awaitable<KeyValue> seek(const silkworm::ByteView& key) override {
+    asio::awaitable<KeyValue> seek(silkworm::ByteView key) override {
```

Comment:
why is passing by reference removed?

Reply:
because it is causing issue #186 with temporary actual argument `silkworm::Bytes{}` lifetime when this coroutine is activated by means of `asio::co_spawn`